Orange/data/table.py

```diff
@@ -1286,6 +1075,7 @@ def _get_matrix(M, cachedM, col):
 
         return distributions
 
+    # TODO: move this to contingency.py and use pandas instead if this code
```

Comment:
Move this there. Then replace code. Don't worry, I fully intend to replace this code, this is just a reminder for what to do.